### PR TITLE
Remove gradients and improve text contrast.

### PR DESCRIPTION
**Pull request by jrobbins**
This is the second of three CLs to refresh the look of chromestatus.

In this CL:
+ Remove gradient backgrounds, and instead use white or very light grey.
+ Use a darker font color for better contrast.
+ Simplify the footer (it will be further simplified when I move login info into the header in my next CL)
+ Stop using all uppercase for milestone markers

### CODE REVIEW

File: static/sass/features/launch.scss [hunk 1]
```diff
@@ -12,7 +12,6 @@
     margin-bottom: 15px;
 
     > div {
-      @include gradient-bar-bg;
       background: rgba(255,255,255,0.8);
```
Comment (liyangguang) [inline static/sass/features/launch.scss:15]: should this to changed to one of the variables as well?

File: static/sass/main.scss [hunk 2]
```diff
@@ -120,24 +121,20 @@ header {
...
+      color: $nav-link-color;
       white-space: nowrap;
 
-      &:active {
-        position: relative;
-        top: 1px;
-        left: 1px;
-        box-shadow: 3px 3px 4px $bar-shadow-color;
+      &:hover {
+	color: black;
```
Comment (liyangguang) [inline static/sass/main.scss:136]: nit: indentation

File: templates/footer.html [hunk 3]
```diff
@@ -1,16 +1,17 @@
...
+        >Privacy</a>
     <a href="https://docs.google.com/document/d/1jrSlM4Yhae7XCJ8BuasWx71CvDEMMbSKbXwx7hoh1Co/edit?pli=1"
        target="_blank"
-       >About</a> |
+       >About</a>
     {% if user %}
     <a href="/settings"
-       >Settings</a> |
+       >Settings</a>
       <b>{{user.email}}</b>
```
Comment (liyangguang) [inline templates/footer.html:15]: nit: indentation to make it obvious they are all siblings.

File: static/sass/main.scss [hunk 4]
```diff
@@ -202,21 +196,27 @@ header {
...
+  padding: 8px;
   bottom: 0;
   left: 0;
   right: 0;
   z-index: 3;
 
   div {
     margin-top: $content-padding / 4;
+
+    a + a, a + b, b + a {
```
Comment (liyangguang) [inline static/sass/main.scss:216]: `& > * + *`?

**Comment by jrobbins** [comment]
Screenshots:
+ Feature list: https://screenshot.googleplex.com/AWxByPtyHFr9mS3
 - With feature cards, app header, app footer, and page background color
+ Releases timeline page: https://screenshot.googleplex.com/3NodaujHfB3ZFsG
+ Samples page: https://screenshot.googleplex.com/5SgvHw27kKF69Ju
+ Metrics page: https://screenshot.googleplex.com/bS9iQ2H6yCzJ4xp
 (my dev server has no metrics to display)

File: templates/footer.html [hunk 5]
```diff
@@ -1,16 +1,17 @@
 <footer>
-  <p>Except as otherwise noted, the content of this page under <a href="https://creativecommons.org/licenses/by/2.5/">CC Attribution 2.5</a> license. Code examples are <a href="https://github.com/GoogleChrome/samples/blob/gh-pages/LICENSE">Apache-2.0</a>.</p>
   <div>
-    <a href="https://groups.google.com/a/chromium.org/forum/#!newtopic/blink-dev"
-       >File content issue</a> |
-    <a href="https://github.com/GoogleChrome/chromium-dashboard/issues"
-       >File an issue</a> |
+     <a href="https://groups.google.com/a/chromium.org/forum/#!newtopic/blink-dev"
```
Comment (liyangguang) [inline templates/footer.html:3]: Is it intended to have some links target="_blank", some not?

**Review by liyangguang — APPROVED** [review]

File: static/sass/features/launch.scss (hunk 1 above)
Comment (jrobbins) [reply static/sass/features/launch.scss:15]: Good catch.  Done.  

Screenshot: https://screenshot.googleplex.com/5mv5m457pUGhXhB

File: templates/footer.html (hunk 3 above)
Comment (jrobbins) [reply templates/footer.html:15]: Fixed.

File: templates/footer.html (hunk 5 above)
Comment (jrobbins) [reply templates/footer.html:3]: Added.

File: static/sass/main.scss (hunk 4 above)
Comment (jrobbins) [reply static/sass/main.scss:216]: Much better.  I should have that one memorized.

File: static/sass/main.scss (hunk 2 above)
Comment (jrobbins) [reply static/sass/main.scss]: This was a tab.  I replaced it with spaces.

**Pull request merged**